src/physics/utilities/solver_config.hpp

```diff
@@ -25,7 +25,8 @@ enum class OutputType
 {
   GLVis,
   ParaView,
-  VisIt
+  VisIt,
```

Comment:
Would we ever want a VisIt output w/o going through Sidre? Seems like we should just hide our use of Sidre under the VisIt output type

Reply:
`VisitDataCollection` is probably more stable than `MFEMSidreDataCollection`, so it may be worth having it as an option as we tune SidreDC.

Reply:
I agree with @joshessman-llnl . I would expect a fair amount of churn in the `MFEMSidreDataCollection`, so it's nice to have a stable option.

Reply:
It's more of a philosophical concern.. `sidre` is not an output format.  As long as its temporary its fine but `SidreVisIt` is closer to the truth.